GTFS.DB.PostgreSQL/PostgreSQLGTFSFeedDB.cs

```diff
@@ -92,11 +92,25 @@ private void RebuildDB()
 
             // Alter existing tables, if their structure is incorrect, for backwards compatibility
             //  1. add agency_email column to agency
-            if (!ColumnExists("agency", "agency_email")) this.ExecuteNonQuery("ALTER TABLE agency ADD COLUMN agency_email TEXT;");
+            if (!ColumnExists("agency", "agency_email"))
+            {
+                this.ExecuteNonQuery("ALTER TABLE agency ADD COLUMN agency_email TEXT;");
+            }
             //  2. add passenger_boarding column to stop_time
-            if (!ColumnExists("stop_time", "passenger_boarding")) this.ExecuteNonQuery("ALTER TABLE stop_time ADD COLUMN passenger_boarding INTEGER;");
+            if (!ColumnExists("stop_time", "passenger_boarding"))
+            {
+                this.ExecuteNonQuery("ALTER TABLE stop_time ADD COLUMN passenger_boarding INTEGER;");
+            }
             //  3. add passenger_alighting column to stop_time
-            if (!ColumnExists("stop_time", "passenger_alighting")) this.ExecuteNonQuery("ALTER TABLE stop_time ADD COLUMN passenger_alighting INTEGER;");
+            if (!ColumnExists("stop_time", "passenger_alighting"))
+            {
+                this.ExecuteNonQuery("ALTER TABLE stop_time ADD COLUMN passenger_alighting INTEGER;");
+            }
+            //  4. add agency_id column to fare_attribute
+            if (!ColumnExists("fare_attribute", "agency_id"))
```

Comment:
need to add it to the table CREATE TABLE IF NOT EXISTS at line 68 as well

Reply:
Yeah I guess this will always add it if it's not there but should probably add it there as well